src/main/java/bio/terra/service/filedata/azure/util/BlobCrl.java

```diff
@@ -0,0 +1,142 @@
+package bio.terra.service.filedata.azure.util;
+
+import com.azure.storage.blob.BlobContainerClient;
+import com.azure.storage.blob.models.BlobProperties;
+import java.net.URL;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import org.apache.commons.lang.StringUtils;
+
+/**
+ * High-level API that enables copy, delete, and get blob properties operations on Azure Blob
+ * storage.
+ */
+public class BlobCrl {
```

Comment:
Correct! And also that is reason, why I put these classes under `util`. The expectation is that they will part of component and how this will get wire up will ready depend of the scenario. We should discuss the use cases, specifically how creds are expected to be provided in user flow to determine what could be driven by config vs some sort of user/app input.